### DISCUSSION TITLE
Dynamic IPv6 Prefixes

### DISCUSSION THREAD

**Discussion by FloMiau** [netbox-community/netbox#5664] [General]
Hey there,

I want to do my network documentation in Netbox but my ISP at home is providing me at home with a dynamic IPv6 prefix. With IPv4 I have NAT, that solves that for me.

What should I do?
My first thought was to use a dummy prefix but maybe there is a better way. 

Thanks.
Flo

**Comment by candlerb** [comment]
I think that's the right way.  You can use a prefix under `fd00::/8` for your Netbox documentation, on the understanding that it will be replaced by the real one when you're online.
